go/engine/login_provision.go

```diff
@@ -566,6 +566,10 @@ func (e *LoginProvision) syncedPGPKey(ctx *Context) (libkb.GenericKey, error) {
 	return unlocked, nil
 }
 
+func (e *LoginProvision) noSyncedKeyMsg() string {
+	return fmt.Sprintf("ERROR!\n\nSorry, you have a PGP private key that Keybase doesn't have a copy of. You need to prove you're you. Any of these will unlock this computer:\n\n1. reset your account and start fresh: https://keybase.io/%s and click \"reset my keys\" in the gear\n2. install GPG and put your PGP private key on this machine and try again\n3. go back and provision with another device or paper key\n\n", e.user.GetName())
```

Comment:
Can we move this output to `client/`? I try to keep U/I-related stuff there as much as possible.  We can import/export the error type and then switch on it...